recipes/default.rb

```diff
@@ -110,15 +110,6 @@
 end
 node.override['hive2']['user'] = hiveUser
 
-sqoopUser = "sqoop"
```

Comment:
I suppose this variable must be used somewhere further to template `core-site.xml`